sdk/metricsadvisor/Azure.AI.MetricsAdvisor/api/Azure.AI.MetricsAdvisor.netstandard2.0.cs

```diff
@@ -744,9 +744,9 @@ public partial class DataFeed
         public DataFeed() { }
         public Azure.AI.MetricsAdvisor.Models.DataFeedAccessMode? AccessMode { get { throw null; } set { } }
         public string ActionLinkTemplate { get { throw null; } set { } }
-        public System.Collections.Generic.IList<string> AdministratorEmails { get { throw null; } }
+        public System.Collections.Generic.IList<string> Administrators { get { throw null; } }
```

Comment:
Removing the `Emails` suffixes because we now know that these properties can also hold IDs (when AAD authentication is used).